src/index.html

```diff
@@ -11,12 +11,58 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    />
     <link
+      href="https://fonts.googleapis.com/css2?family=Jost:wght@400;500&family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header_logo" href="/">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#sumsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" data-qa="hover"
+              href="#laptops-and-computers">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting requires a 2-space offset. Also, don't add blank lines between parent and child elements.